internal/cli/atlas/atlas_darwin.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build darwin
+// +build darwin
```

Comment:
[fyi-non blocking] the name of the file already does this, you don't need to add build tags if the file ends on the name of the GOOS you're targetting

Reply:
yes I know but I find the build tags easier to notice and I didn’t know how to name the files 😅